packages/core/src/components/ResponseErrorPanel/ResponseErrorPanel.tsx

```diff
@@ -107,12 +109,21 @@ const ResponseErrorList = ({
           </ListItem>
         </>
       )}
+      {actions && (
+        <>
+          <Divider component="li" className={classes.divider} />
+          <ListItem alignItems="flex-start">{actions}</ListItem>
+        </>
+      )}
     </List>
   );
 };
 
 type Props = {
   error: Error;
+  defaultExpanded?: boolean;
+  title?: string;
+  actions?: React.ReactNode;
```

Comment:
Not a huge fan of stacking more and more props onto this one if we don't have to - long term it leads to complex beahvior and interfaces. Would it be possible to instead render your own `WarningPanel` containing a `ResponseErrorDetails` and then actions below that? Composition is often better